### DISCUSSION TITLE
Unable to revoke from dashboard

### DISCUSSION THREAD

**Discussion by ash06dec** [polymind-inc/acmebot#528] [Q&A] [answered]
I'm unable to revoke the certificate from the dashboard. 

Creation & renew part are working as expected. 

am I missing anything or doing wrong ?

**Comment by shibayan** [comment, marked as answer]
Revoke is the process of updating the Let's Encrypt certificate revocation list; the status will not change on Key Vault, but the client performing the OCSP check should see the revoked status.

**Reply by ash06dec** [reply]
After the revoke, still I can see the certificate is in use & shows secure..

Expected result from web application, "Not secure".

**Reply by shibayan** [reply]
It is a browser, mostly Chromium-based issue. crt.sh can be used to check revocation at OCSP.

https://crt.sh/

We believe it will take some time before each browser actually treats it as revoked.
